setup.py

```diff
@@ -85,7 +85,7 @@ def get_package_data(parent, roots):
 
 setup(
     name="pyfarm.agent",
-    version="0.8.3",
+    version="0.8.3.1",
```

Comment:
0.8.3 is not on PyPi or tagged yet.  Is there any reason we can't tag and release 0.8.3?  This would also keep the versioning scheme consistent across packages and between previous and future versions.